cpp/include/cudf/strings/detail/utf8.hpp

```diff
@@ -29,16 +29,45 @@ using char_utf8 = uint32_t;  ///< UTF-8 characters are 1-4 bytes
 namespace strings {
 namespace detail {
 
+/**
+ * @brief This will return true if passed a continuation byte of a UTF-8 character.
+ *
+ * @param byte Any byte from a valid UTF-8 character
+ * @return true if this is not the first byte of the character
+ */
+constexpr bool is_utf8_continuation_char(uint8_t byte)
```

Comment:
This will ripple out to other code if I change the signature of `is_begin_utf8_char()`.  I'd rather just change "byte" to "chr" or something for now. I wish std::byte was easier to use, I really want to like it...

Reply:
yeah, I can completely understand that and I don't have an issue with going that route. It just seems wrong to call it byte. I can also argue against calling it char, but we have to pick the hills on which to die and I choose function names being consistent with variable names. Thanks for placating me.

Reply:
C++ has too many types that can manage the same content. And now we SWE developers as a precious resource have to burn our precious neurons arguing which one to pick :joy: